pdp/assignment.go

```diff
@@ -453,13 +463,26 @@ func (a AttributeAssignment) Serialize(ctx *Context) (string, string, string, er
 }
 
 func (a AttributeAssignment) String() string {
-	name, valueType, value, err := a.Serialize(nil)
+	name, valueType, value, err := a.Serialize(emptyCtx)
 	if err != nil {
 		return err.Error()
 	}
 	return fmt.Sprintf("(%s)%s:%s", valueType, name, value)
 }
 
+// MarshalJSON satisfies Marshaler interface
```

Comment:
Please mention here in the comment that it works for an assignment expression which righthand part doesn't depend on context. Examples of such expression are a value, constant expression, selector which doesn't rely on attributes or local content.